doc/man.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
```

Comment:
Do you want a specific target to generate it, or generate it systematically when compiling the executable ? Just asking to know if we need to implement a check for the existence of man2html or similar converters. Also we would still need to commit the change to the html (if any) manually - but this cannot  really be helped.